client/blocks/product-plan-overlap-notices/docs/example.jsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SitesDropdown from 'components/sites-dropdown';
+import ProductPlanOverlapNotices from '../';
+
+const jetpackPlans = [
+	'jetpack_personal',
+	'jetpack_personal_monthly',
+	'jetpack_premium',
+	'jetpack_premium_monthly',
+	'jetpack_business',
+	'jetpack_business_monthly',
+];
+
+const jetpackProducts = [
+	'jetpack_backup_daily',
+	'jetpack_backup_daily_monthly',
+	'jetpack_backup_realtime',
+	'jetpack_backup_realtime_monthly',
+];
+
+class ProductPlanOverlapNoticesExample extends Component {
+	state = {
+		siteId: 0,
+	};
+
+	render() {
+		return (
+			<Fragment>
```

Comment:
I think this `<Fragment>` is redundant here. We've got a `<div>` as an only child of it.

Reply:
Nice catch - fixed in https://github.com/Automattic/wp-calypso/pull/37513/commits/eba7c71f59ec4b079395e3c6c2b3f8fb36ab85ad